libfreeipmi/include/freeipmi/cmds/ipmi-lan-cmds.h

```diff
@@ -221,6 +221,38 @@ extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_rmcpplus_messag
 extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_rmcpplus_messaging_cipher_suite_privilege_levels_rs;
 extern fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold_rs;
 
```

Comment:
I know you probably left this for now for your own use, but eventually remove space, to "chunk" all of these packets together in this header